Cabal/doc/installing-packages.rst

```diff
@@ -624,21 +633,20 @@ package:
     Installation directory for the configuration files.
 
     In the simple build system, *dir* may contain the following path
-    variables: ``$prefix``, ``$bindir``, ``$libdir``, ``$libsubdir``,
+    variables: ``$prefix``, ``$bindir``, ``$libdir``, ``$commonlibdir``,
     ``$pkgid``, ``$pkg``, ``$version``, ``$compiler``, ``$os``,
     ``$arch``, ``$abi``, ``$abitag``
 
 In addition the simple build system supports the following installation
 path options:
 
-.. option:: --libsubdir=dir
+.. option:: --commonlibdir=dir
 
     A subdirectory of *libdir* in which libraries are actually
     installed. For example, in the simple build system on Unix, the
-    default *libdir* is ``/usr/local/lib``, and *libsubdir* contains the
-    package identifier and compiler, e.g. ``mypkg-0.2/ghc-6.4``, so
-    libraries would be installed in
-    ``/usr/local/lib/mypkg-0.2/ghc-6.4``.
+    default *libdir* is ``/usr/local/lib``, and *commonlibdir* contains the
+    ABI, e.g. ``x86_64-linux-8.0.1``, so libraries would be installed in
+    ``/usr/local/lib/x86_64-linux-8.0.1``.
```

Comment:
It is recommended that a single, common directory to be used to store all installed libraries (as opposed to using \`\`$pkgid\`\` or similar variables to create a directory per installed library), as this helps reduce the size of rpath in executables built against dynamic libraries.  See https://github.com/haskell/cabal/pull/3982 for more details.